hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/index.md

```diff
@@ -1671,6 +1671,25 @@ To enable this feature within S3A, configure the `fs.s3a.requester.pays.enabled`
 </property>
 ```
 
+## <a name="storage_classes"></a>Storage Classes
+
+Amazon S3 offers a range of [Storage Classes](https://aws.amazon.com/s3/storage-classes/) 
+that you can choose from based on behavior of your applications. By using the right 
+storage class, you can reduce the cost of your bucket.
+
+S3A uses Standard storage class for PUT object requests by default, which is suitable for 
+general use cases. To use a specific storage class, set the value in `fs.s3a.storage.class` property to
+the storage class you want.
+
+```xml
+<property>
+    <name>fs.s3a.storage.class</name>
+    <value>intelligent_tiering</value>
+</property>
+```
+
+Please note that S3A does not support reading from archive storage classes at the moment.
```

Comment:
can you add that if someone tries this, they will get access denied with InvalidObjectState. or add the error/trace to the troubleshooting section where the other exceptions are.

Reply:
got it